tests/Microsoft.Azure.Batch.SoftwareEntitlement.Tests/ServerOptionBuilderTests.cs

```diff
@@ -77,5 +77,13 @@ public void Build_WithEmptyAudience_HasDefaultValueForAudience()
             var options = ServerOptionBuilder.Build(_commandLine);
             options.Value.Audience.Should().NotBeNullOrEmpty();
         }
+
+        [Fact]
+        public void Build_WithEmptyIssuer_HasDefaultValueForIssuer()
+        {
+            _commandLine.Issuer = string.Empty;
+            var options = ServerOptionBuilder.Build(_commandLine);
+            options.Value.Issuer.Should().NotBeNullOrEmpty();
```

Comment:
Given the test name should this assert that there is a particular value rather than just 'non-empty'?